### PR TITLE
Support for python stack analysis

### PR DESCRIPTION
**Pull request by samuzzal-choudhury**

### CODE REVIEW

**Comment by samuzzal-choudhury** [comment]
https://gist.github.com/samuzzal-choudhury/d8334a6a75e8abaff1fc8f7b093237b7

File: cucoslib/workers/recommender.py [hunk 1]
```diff
@@ -135,30 +135,31 @@ def get_topmost_ref_stack(self, list_packages):
...
+                'gremlin': str_gremlin,
+                'bindings': {
+                    'sname': sname
+                }
             }
-        }
-        json_response = self.execute_gremlin_dsl(payload)
-        if json_response is not None:
-            return self.get_response_data(json_response, data_default=[])
+            json_response = self.execute_gremlin_dsl(payload)
```
Comment (tuxdna) [inline cucoslib/workers/recommender.py]: To confirm, is this only indentation change ?

File: cucoslib/manifests.py [hunk 2]
```diff
@@ -58,7 +58,12 @@ def xml_validator(data):
         return False
     return True
 
+def python_validator(data):
+    """Very simple Python requirements.txt validator."""
+    # TODO
```
Comment (tuxdna) [inline cucoslib/manifests.py]: This is marked as TODO.

File: cucoslib/workers/recommender.py (hunk 1 above)
Comment (samuzzal-choudhury) [inline cucoslib/workers/recommender.py]: @tuxdna that's right. [This condition](https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/10/files#diff-e27424ca7473f0836da5680fa3e9d08dR138) has been added to handle an error condition when there is no reference stack in the system.

Comment (samuzzal-choudhury) [inline cucoslib/workers/recommender.py]: @tuxdna everything inside the above condition was already present in the code.

File: cucoslib/manifests.py (hunk 2 above)
Comment (samuzzal-choudhury) [inline cucoslib/manifests.py]: @tuxdna added the logic. Please review.

**Comment by tuxdna** [comment]
LGTM

**Pull request merged**